src/bench/rpc_mempool.cpp

```diff
@@ -12,7 +12,7 @@
 static void AddTx(const CTransactionRef& tx, const CAmount& fee, CTxMemPool& pool) EXCLUSIVE_LOCKS_REQUIRED(cs_main, pool.cs)
 {
     LockPoints lp;
-    pool.addUnchecked(CTxMemPoolEntry(tx, fee, /* time */ 0, /* height */ 1, /* spendsCoinbase */ false, /* sigOpCost */ 4, lp));
+    pool.addUnchecked(CTxMemPoolEntry(tx, fee, /*_nTime=*/0, /*_entryHeight=*/1, /*spendsCoinbase=*/false, /*nSigOpsCost=*/4, lp));
```

Comment:
It's unfortunate in these cases that these argument names are so ugly (e.g. `_nTime` doesn't just have an underscore, but also the long deprecated syntax)

Reply:
Happy to review a follow-up to fixup the name of the args, or should I do it here?

Reply:
Not checking them makes it harder to catch bugs, like #22979. So I think this is overall a good idea.

Reply:
Fixed in #23054